docs/docs/10-users/20-getting-started/70-conclusion.md

```diff
@@ -2,10 +2,8 @@
 
 🎉 Congratulations! You have successfully created a Renku project, connected data and code, launched a session, and shared your project with collaborators. 
 
-You can find this project available here: https://renkulab.io/v2/projects/renku-team/zurich-air-quality-analysis
+You can find this project available [here](https://renkulab.io/p/renku-team/zurich-air-quality-analysis).
```

Comment:
Note: "here" is bad practice for links. See: https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text

Prefer giving a short title to links:

```suggestion
You can find this project available here: [Zurich Air Quality Analysis](https://renkulab.io/p/renku-team/zurich-air-quality-analysis).
```

Reply:
Thanks!